lib/std/os/bits/darwin.zig

```diff
@@ -1554,3 +1554,186 @@ pub const rlimit = extern struct {
 pub const SHUT_RD = 0;
 pub const SHUT_WR = 1;
 pub const SHUT_RDWR = 2;
+
+// Term
+pub const VEOF = 0;
+pub const VEOL = 1;
```

Comment:
Could we constrain the types rather than leave them hanging at `usize`?

Reply:
Truth is I don't know what these variables are used for (nor which syscall is accepting them). I could make an extern enum, but then which numeric type should the enum be? In the original code they are macro definitions.

Reply:
Oh sure. I still think in Zig we are in the privileged position to be *always* able to specify the precise bit-width of a primitive type, and I believe we should always strive to do this as this can uncover potential bugs when something changes in the future between versions. Unlike C/C++, we don't have to resort to completely unhygienic preproc macros!

Having said that, I'm fine leaving it as-is for now, but would really welcome a subsequent PR that tries attaching a type to all these. :-)

Reply:
Yes, I think somebody that actually knows what this stuff does should do a cleanup pass in the future, I'm only trying to make my chat application to build on macos :)

Reply:
Chances are you are the expert on these now :D